zipkin-server/pom.xml

```diff
@@ -230,6 +223,18 @@
       <version>2.4.0</version>
       <scope>test</scope>
     </dependency>
+
+    <!-- Micrometer and Prometheus Registry-->
+    <dependency>
+      <groupId>io.micrometer</groupId>
+      <artifactId>micrometer-spring-legacy</artifactId>
+      <version>${micrometer.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.micrometer</groupId>
+      <artifactId>micrometer-registry-prometheus</artifactId>
+      <version>${micrometer.version}</version>
```

Comment:
prometheus was formerly optional dep. I'd be careful on this and also make sure if this is to replace zipkin-autoconfigure-metrics-prometheus that the format especially of collector metrics is exactly the same